.github/workflows/docs-pr.yml

```diff
@@ -20,7 +20,7 @@ jobs:
       init-fail-on-error: true
       intersphinx-links: |
         community_aws:https://ansible-collections.github.io/community.aws/branch/main/
-        ansible_devel:https://docs.ansible.com/ansible-core/devel/
+        ansible_stable:https://docs.ansible.com/ansible-core/2.17/
       artifact-name: ${{ github.event.repository.name }}_validate_docs_${{ github.event.pull_request.head.sha }}
```

Comment:
`ansible-ref: devel` does fix too. I would probably go with `ansible-ref: devel`, to avoid sticking to a specific ansible-core version. Any thought @tremble?